docs/getting-started-guides/ubuntu.md

```diff
@@ -246,6 +247,42 @@ the latter one could start it again.
 4. You can also customize your own settings in `/etc/default/{component_name}`.
 
 
+### Upgrading a Cluster
+
+If you already have a kubernetes cluster, and want to upgrade to a new version,
+you can use following command in cluster/ directory to update the whole cluster or a specified node to a new version.
+
+```console
+KUBERNETES_PROVIDER=ubuntu ./kube-push.sh [-m|-n <node id>] <version>
+```
+
+It can be done for all components (by default), master(`-m`) or specified node(`-n`).
+If the version is not specified, the script will try to use local binaries.
```

Comment:
Support for local binaries would be implemented by #13900 
This pr should only be merged after that to get rid of potential inconsistency, or you'd better kindly remove this as well.